diff --git a/libcontainer/configs/validate/validator.go b/libcontainer/configs/validate/validator.go
index cbbba9a0..1eb3fca8 100644
--- a/libcontainer/configs/validate/validator.go
+++ b/libcontainer/configs/validate/validator.go
@@ -52,7 +52,7 @@ func (v *ConfigValidator) Validate(config *configs.Config) error {
 	return nil
 }
 
-// rootfs validates if the rootfs is an absolute path and is not a symlink
+// rootfs validates if the rootfs is an absolute path
 // to the container's root filesystem.
 func (v *ConfigValidator) rootfs(config *configs.Config) error {
 	if _, err := os.Stat(config.Rootfs); err != nil {
@@ -65,11 +65,8 @@ func (v *ConfigValidator) rootfs(config *configs.Config) error {
 	if err != nil {
 		return err
 	}
-	if cleaned, err = filepath.EvalSymlinks(cleaned); err != nil {
-		return err
-	}
 	if filepath.Clean(config.Rootfs) != cleaned {
-		return fmt.Errorf("%s is not an absolute path or is a symlink", config.Rootfs)
+		return fmt.Errorf("%s is not an absolute path", config.Rootfs)
 	}
 	return nil
 }
